packages/shared/src/components/widgets/ReferralWidget.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement } from 'react';
+import { Button, ButtonSize } from '../buttons/Button';
+import { TextField } from '../fields/TextField';
+import { link } from '../../lib/links';
+import { useShareOrCopyLink } from '../../hooks/useShareOrCopyLink';
+import { labels } from '../../lib';
+import { AnalyticsEvent, TargetId } from '../../lib/analytics';
+import ReferralSocialShareButtons from './ReferralSocialShareButtons';
+
+const ReferralWidget = ({ url }: { url: string }): ReactElement => {
+  const inviteLink = url || link.referral.defaultUrl;
+  const [, onShareOrCopyLink] = useShareOrCopyLink({
+    text: labels.referral.generic.inviteText,
+    link: inviteLink,
+    trackObject: () => ({
+      event_name: AnalyticsEvent.CopyReferralLink,
+      target_id: TargetId.ProfilePage,
+    }),
+  });
+  const onShareClick = () => {
+    onShareOrCopyLink();
+  };
+
+  return (
+    <div
+      data-testid="referral-widget"
+      className="flex laptopL:absolute flex-col tablet:flex-row laptopL:flex-col flex-wrap justify-between p-4 laptopL:m-6 mt-6 laptopL:mt-0 mb-4 laptopL:max-w-widget laptopL:h-auto rounded-2xl border border-theme-divider-tertiary laptopL:left-[40rem] max-w-fit bg-theme-bg-primary"
+    >
+      <h3 className="mb-2 font-bold typo-title3">Invite friends</h3>
+      <p className="contents text-theme-label-secondary typo-callout">
```

Comment:
Can you elaborate why do need this class? Seems to be not available on tailwindcss.
```suggestion
      <p className="text-theme-label-secondary typo-callout">
```